src/probability/independence.lean

```diff
@@ -342,4 +342,30 @@ lemma indep_sets.indep_set_of_mem (hs : s ∈ S) (ht : t ∈ T) (hs_meas : measu
 
 end indep_set
 
+section indep_fun
+
+variables {α β β' γ γ' : Type*} [measurable_space α] [measurable_space β] [measurable_space β']
+variables [measurable_space γ] [measurable_space γ'] {μ : measure α}
```

Comment:
The measurable spaces would be better with implicit arguments:
```suggestion
variables {α β β' γ γ' : Type*} {mα : measurable_space α} {mβ : measurable_space β}
  {mβ' : measurable_space β'} {mγ : measurable_space γ} {mγ' : measurable_space γ'}
  {μ : measure α}

include mα mβ
```
(you'll need to add an `include` for the others where you need them)

The reason is that we can then use the lemmas for `measurable_space` arguments which are not the one found in the typeclass search and it will still work, because lean is able to get the `measurable_space` from other arguments like `indep_fun`.

Reply:
OK, makes sense!